scripts/prepare_remote_test.py

```diff
@@ -0,0 +1,146 @@
+
+
+class DropletSetup(object):
+    def __init__(self, token, size, region):
+        super(DropletSetup, self).__init__()
+        self.token = token
+        self.size = size
+        self.region = region
+        self.ssh_client = paramiko.SSHClient()
+        self.ssh_client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+        self.ssh_key = None
+        self.digital_ocean_ssh_key = None
+
+        self._generate_ssh_key()
+        self.droplet = digitalocean.Droplet(
+            token=self.token,
+            name=DROPLET_NAME,
+            region=self.region,
+            image='ubuntu-16-04-x64',
+            size_slug=self.size,
+            ssh_keys=[self.digital_ocean_ssh_key.id]
+        )
+
+    @staticmethod
+    def _print_info(message):
+        print('[{timestamp}]\t{message}'.format(timestamp=datetime.now().isoformat(), message=message))
+
+    def _execute_command(self, command):
+        self._print_info('executing {command}'.format(command=command))
+        _, _, std_err = self.ssh_client.exec_command(command)
+
+        #for line in std_out.readlines():
+        #    print(line.replace('\n', ''))
+
+        for line in std_err.readlines():
+            print(line.replace('\n', ''))
+            raise Exception('Script execution failed')
+
+    def _generate_ssh_key(self):
+        self._print_info('generating ssh key')
+        self.ssh_key = paramiko.rsakey.RSAKey.generate(2048, str(uuid.uuid4()))
+
+        self._print_info('create ssh key on DigitalOcean')
+        self.digital_ocean_ssh_key = digitalocean.SSHKey(
+            token=self.token,
+            name=SSH_KEY_NAME,
+            public_key='ssh-rsa {key}'.format(key=str(self.ssh_key.get_base64()))
+        )
+
+        self.digital_ocean_ssh_key.create()
+
+    def create_droplet(self):
+        self._print_info('creating droplet')
+        self.droplet.create()
+
+        self._print_info('waiting for droplet to be ready')
+        self._wait_for_droplet()
+
+    def _wait_for_droplet(self):
+        actions = self.droplet.get_actions()
+        for action in actions:
+            if action.status == DROPLET_STATUS_COMPLETED:
+                self.droplet.load()
+                return
+
+        self._wait_for_droplet()
+
+    def connect(self):
+        self._print_info('connecting to droplet')
+        try:
+            self.ssh_client.connect(
+                hostname=self.droplet.ip_address,
+                username='root',
+                allow_agent=True,
+                pkey=self.ssh_key
+            )
+        except Exception as exc:
+            self._print_info(str(exc))
+            self._print_info('reconnecting')
+            sleep(3)
+            self.connect()
+
+    def install_rebirthdb(self):
+        self._print_info('getting rebirthdb')
+        self._execute_command('source /etc/lsb-release && echo "deb https://dl.bintray.com/floydkots/apt $DISTRIB_CODENAME main" | tee /etc/apt/sources.list.d/rebirthdb.list')
```

Comment:
This is looking good! I'd like to suggest that to allow for flexibility, we should get the bintray username from the environment.

It would look something like:
```python
self._execute_command('source /etc/lsb-release && echo "deb https://dl.bintray.com/$BINTRAY_USERNAME/apt $DISTRIB_CODENAME main" | tee /etc/apt/sources.list.d/rebirthdb.list')
self._execute_command('wget -qO- https://dl.bintray.com/$BINTRAY_USERNAME/keys/pubkey.gpg | apt-key add -')
```
The repository structures will however remain fairly constant. So, once we have the packages under our **rebirthdb** bintray organization, it will be a simple matter of changing the value of the travis (or wherever the script is executed) environment variable from `floydkots` to `rebirthdb`.

Reply:
I will fix it today